week-4/Homework/mandatory/4-groceries.js

```diff
@@ -28,8 +28,20 @@ Exercise 1:
   Then use console.log() to print out the list.
 */
 // Gather all week item names into this array
+
+// Solution 1
 let weeklyGroceriesToBuy = [];
+for (let key in weeklyMealPlan) {
+    let values = weeklyMealPlan[key]
+    for (let item of values) {
+        weeklyGroceriesToBuy.push(item)
+    }
+}
+console.log(weeklyGroceriesToBuy)
+
+//Solution 2 
 
+// let weeklyGroceriesToBuy = Array.from([].concat(...Object.values(weeklyMealPlan)))
```

Comment:
this is brilliant, however, it using a couple of methods from ES6. let's keep it simple join/concat?

Reply:
added as solution 3